proxies/live/apiproxy/policies/Quota.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Quota name="Quota">
+    <DisplayName>Quota</DisplayName>
+    <Allow count="300"/>
```

Comment:
Stick a TODO or FIXME in here so we come back to this? (And read it out of the API Product being used)